data/src/update.rs

```diff
@@ -59,6 +59,20 @@ pub struct Version {
     pub patch: u8,
     pub pre: Option<PreRelease>,
 }
+impl Display for Version {
```

Comment:
The ? operator is your friend in these sorts of functions.  Matter of taste though.

Reply:
Good catch. Will change.